server/api/repo.go

```diff
@@ -114,6 +114,31 @@ func PostRepo(c *gin.Context) {
 		sig,
 	)
 
+	// find org of repo
```

Comment:
wee need to check if it's not an org but user repo

Reply:
Why? Its intended to also save orgs for users as it should be possible to set for exp secrets for your user-org as well.